hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1516,6 +1516,9 @@ public void start() throws IOException {
     metrics.setNumFiles(metadataManager
         .countEstimatedRowsInTable(metadataManager.getFileTable()));
 
+    // Snapshot metrics
+    updateActiveSnapshotMetrics();
```

Comment:
instead of invoking this in 3 places, lets invoke in instantiateServices() right after the call to:
```
    omSnapshotManager = new OmSnapshotManager(this);
```